### ISSUE TITLE
When opt-in to markdown links check, the cloned DscResource.Tests is also checked

### ISSUE THREAD

**Issue by sergey-s-betke** [PowerShell/DscResource.Tests#322]
#### Details of the problem, bug, or enhancement

When opt-in to markdown links check, the cloned DscResource.Tests is also spell-checked

#### Verbose logs showing the problem (if applicable)

https://ci.appveyor.com/project/sergey-s-betke/xitghyperv/builds/24480593

**Comment by PlagueHO** [comment]
Good catch @sergey-s-betke - thanks for raising this! It should be an easy fix. Labelled it as good first issue.
